### PR TITLE
jeffturner_homework4

### PR DESCRIPTION
**Pull request by jeffreymt1**
Completion of Homework4

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	$('.readless a').hide();
+	$('.readmore').click(articleOpen);
+	$('.readless').click(articleClose);
+	$('.learnmore').click(learnMoreOpen);
+
+
+	// article read more and read less
+	function articleOpen() {
+		$('#show-this-on-click').slideDown();
+		$('.readmore a').hide();
```
Comment (erikawithak) [inline js/index.js]: This is absolutely correct, just keeping in mind by targeting the `a` element for hide/show you are leaving the parent element with the class(es) .readmore/.readless in your markup. You could also just target those directly so you don't have empty elements on your page.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		$('.readless a').show();
+	}
```
Comment (erikawithak) [inline js/index.js]: You left out the `event.preventDefault` which prevents the `a` link element from it's normal behavior of reloading the page. Without this, clicking on each link will just reload the JS and prevent your JS from running.

File: js/index.js (hunk 1 above)
Comment (jeffreymt1) [inline js/index.js]: ^ noted, thanks

File: js/index.js (hunk 2 above)
Comment (jeffreymt1) [inline js/index.js]: ^yup.  I changed the html from "#" to "#/" to prevent the reload of the page, but agree it'd be better to execute via js.

**Pull request closed without merging**